Apps/ValidationTests/Win32/App.cpp

```diff
@@ -270,7 +270,7 @@ LRESULT CALLBACK WndProc(HWND hWnd, UINT message, WPARAM wParam, LPARAM lParam)
             }
             break;
         }
-        case WM_DESTROY:
+        case WM_CLOSE:
         {
             short exitCode = LOWORD(wParam);
```

Comment:
I'm a bit confused by this. Documentation says that the lParam and wParam of both WM_DESTROY or WM_CLOSE are not used. What does this do exactly?